File: cpp/TokoElektronik.cpp
#include <string>
#include <iostream>
using namespace std;
class TokoElektronik{
    protected: //ini supaya atribut parent bisa diakses sama child
        string nama_toko;
        string alamat_toko;
        string notel_toko;

    public:
    TokoElektronik(){
    }
 //Get Set nama toko
    void SETNama(string nama_toko){
        this->nama_toko = nama_toko;
    }
    string getNama() {
        return this->nama_toko;
    }

    //Get Set alamat toko
    void SETAlamat(string alamat_toko){
        this->alamat_toko = alamat_toko;
    }
    string getAlamat() {
        return this->alamat_toko;
    }

    //Get Set nomor telepon
    void SETNotel(string notel_toko){
        this->notel_toko = notel_toko;
    }
     
    string getNotel() {
        return this->notel_toko;
    }

    //Method add data toko elektronik
    void addData(string nama_toko, string alamat_toko, string notel_toko) {
        this->nama_toko = nama_toko;
        this->alamat_toko = alamat_toko;
        this->notel_toko = notel_toko;
    }


    ~TokoElektronik() {

    }
};

File: php/TokoElektronik.php
<?php
class TokoElektronik {
    protected $nama_toko;
    protected $alamat_toko;
    protected $notel_toko;

    public function __construct($nama = "", $alamat = "", $notel = "") {
        $this->nama_toko = $nama;
        $this->alamat_toko = $alamat;
        $this->notel_toko = $notel;
    }

    public function getNama() { return $this->nama_toko; }
    public function getAlamat() { return $this->alamat_toko; }
    public function getNotel() { return $this->notel_toko; }

    public function addData($nama, $alamat, $notel) {
        $this->nama_toko = $nama;
        $this->alamat_toko = $alamat;
        $this->notel_toko = $notel;
    }
}
?>
